src/status_im/feature_flags.cljs

```diff
@@ -14,6 +14,8 @@
    ::settings.wallet-settings           (enabled-in-env? :FLAG_WALLET_SETTINGS_ENABLED)
    ::settings.keypairs-and-accounts     (enabled-in-env?
                                          :FLAG_WALLET_SETTINGS_KEYPAIRS_AND_ACCOUNTS_ENABLED)
+   ::settings.network-settings          (enabled-in-env?
+                                         :FLAG_WALLET_SETTINGS_NETWORK_SETTINGS_ENABLED)
```

Comment:
Hmm I suppose I would prefer `120` characters, but I wonder is there a way to configure zprint for specific namespaces? Maybe we can these configuration namespaces could be `120` by default?

Reply:
I've never seen that feature, but maybe there's a way. The author is very open for feedback because he released a few versions fixing things we reported.

Reply:
If I find the time I'll try to open a GH discussion, full of examples of bad looking Clojure code in this repo and how a human would format them if the character limit was slightly higher.